### ISSUE TITLE
[Suggestion] Changing the rotation of the celestial sphere

### ISSUE THREAD

**Issue by johndelker** [Nettakrim/Spyglass-Astronomy#42]
I'd like to be able to change in which direction the the north pole of the celestial sphere is. In other words, what point the stars rotate around.

**Comment by Nettakrim** [comment]
that is a good idea

**Comment by Nettakrim** [comment]
i assume that would also change the plane the sun and moon are on? otherwise the physics is innacurate

**Comment by johndelker** [comment]
Yes, although technically, the moon doesn't have to be on the same plane as the sun and planets. Earth's moon is ~5 degrees off. Moons that are close-in to their planets often have orbital planes more lined up with the planet's equator, not with the the solar system plane.

**Comment by Nettakrim** [comment]
yeah, ofc in minecraft the moon is strictly opposite the sun, and changing that is too much of a change to vanilla for me to justify it
